src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1741,6 +1741,71 @@ public void GetPartsOfCastExpression(SyntaxNode node, out SyntaxNode type, out S
             return null;
         }
 
+        public SyntaxToken? GetTargetIfControlFlow(SyntaxToken token)
+        {
+            return token.Kind() switch
+            {
+                SyntaxKind.ContinueKeyword => TryGetStartTarget(TryFindContainingLoop(token.Parent)),
+                SyntaxKind.BreakKeyword => TryGetEndTarget(TryFindContainingLoop(token.Parent)),
+                _ => null
+            };
+
+            SyntaxNode? TryFindContainingLoop(SyntaxNode? node)
```

Comment:
this exists... somewhere.  i'll see if i can help you find it.

Reply:
not sure where it is.  but, as an example, 'keyword highlighter' has an understading of what to highlight when you place your caret on a break/continue.

Reply:
Helpers are IsContinuableConstruct and IsBreakableConstruct